### ISSUE TITLE
Can you help me specify the specific dependency version?

### ISSUE THREAD

**Issue by kaixindelele** [Stanford-ILIAD/droc#3]
Can you help me specify the specific dependency version? especially `transformers`.

It would be great if you could help tell me all the versions in req.txt.

**Comment by lihzha** [comment]
Hi, 

The version for `transformers` is 4.33.1. For other dependencies:

| Name                      | Version    | Build         | Channel          |
|---------------------------|------------|---------------|------------------|
| 2captcha-python | 1.2.1 | pypi_0 | pypi |
| _libgcc_mutex | 0.1 | main |
| _openmp_mutex | 5.1 | 1_gnu |
| absl-py | 1.4.0 | pypi_0 | pypi |
| addict | 2.4.0 | pypi_0 | pypi |
| aiodns | 3.0.0 | pypi_0 | pypi |
| aiohttp | 3.8.5 | pypi_0 | pypi |
| aiohttp-retry | 2.8.3 | pypi_0 | pypi |
| aiosignal | 1.3.1 | pypi_0 | pypi |
| aleph-alpha-client | 2.14.0 | pypi_0 | pypi |
| ansi2html | 1.8.0 | pypi_0 | pypi |
| anthropic | 0.2.10 | pypi_0 | pypi |
| anyio | 4.0.0 | pypi_0 | pypi |
| astroid | 2.14.2 | py39h06a4308_0 |
| asttokens | 2.3.0 | pypi_0 | pypi |
| async-timeout | 4.0.3 | pypi_0 | pypi |
| attrs | 23.1.0 | pypi_0 | pypi |
| backcall | 0.2.0 | pyh9f0ad1d_0 | conda-forge |
| backports | 1.0 | pyhd8ed1ab_3 | conda-forge |
| backports.functools_lru_cache | 1.6.5 | pyhd8ed1ab_0 | conda-forge |
| beautifulsoup4 | 4.12.2 | pypi_0 | pypi |
| bert-score | 0.3.13 | pypi_0 | pypi |
| blanc | 0.3.3 | pypi_0 | pypi |
| blis | 0.7.10 | pypi_0 | pypi |
| boto3 | 1.28.40 | pypi_0 | pypi |
| botocore | 1.31.40 | pypi_0 | pypi |
| bottle | 0.12.25 | pypi_0 | pypi |
| ca-certificates | 2023.08.22 | h06a4308_0 |
| cachetools | 5.3.1 | pypi_0 | pypi |
| catalogue | 2.0.9 | pypi_0 | pypi |
| cattrs | 22.2.0 | pypi_0 | pypi |
| certifi | 2023.7.22 | pypi_0 | pypi |
| cffi | 1.15.1 | pypi_0 | pypi |
| charset-normalizer | 3.2.0 | pypi_0 | pypi |
| click | 8.1.7 | pypi_0 | pypi |
| cloudpickle | 2.2.1 | pypi_0 | pypi |
| colorama | 0.4.6 | py39h06a4308_0 |
| colorcet | 3.0.1 | pypi_0 | pypi |
| comm | 0.1.4 | pyhd8ed1ab_0 | conda-forge |
| confection | 0.1.1 | pypi_0 | pypi |
| configargparse | 1.7 | pypi_0 | pypi |
| contourpy | 1.1.0 | pypi_0 | pypi |
| crfm-helm | 0.2.3 | pypi_0 | pypi |
| cycler | 0.11.0 | pypi_0 | pypi |
| cymem | 2.0.7 | pypi_0 | pypi |
| cython | 3.0.2 | pypi_0 | pypi |
| dacite | 1.6.0 | pypi_0 | pypi |
| dash | 2.13.0 | pypi_0 | pypi |
| dash-core-components | 2.0.0 | pypi_0 | pypi |
| dash-html-components | 2.0.0 | pypi_0 | pypi |
| dash-table | 5.0.0 | pypi_0 | pypi |
| datasets | 2.5.2 | pypi_0 | pypi |
| debugpy | 1.6.7 | py39h6a678d5_0 |
| decorator | 5.1.1 | pyhd8ed1ab_0 | conda-forge |
| dill | 0.3.5.1 | pypi_0 | pypi |
| emoji | 2.8.0 | pypi_0 | pypi |
| entrypoints | 0.4 | pyhd8ed1ab_0 | conda-forge |
| exceptiongroup | 1.1.3 | pyhd8ed1ab_0 | conda-forge |
| executing | 1.2.0 | pyhd8ed1ab_0 | conda-forge |
| fastjsonschema | 2.18.0 | pypi_0 | pypi |
| filelock | 3.12.3 | pypi_0 | pypi |
| flask | 2.2.5 | pypi_0 | pypi |
| flatbuffers | 23.5.26 | pypi_0 | pypi |
| fonttools | 4.42.1 | pypi_0 | pypi |
| frozenlist | 1.4.0 | pypi_0 | pypi |
| fsspec | 2023.9.0 | pypi_0 | pypi |
| ftfy | 6.1.1 | pypi_0 | pypi |
| gdown | 4.4.0 | pypi_0 | pypi |
| gin-config | 0.5.0 | pypi_0 | pypi |
| google-api-core | 2.11.1 | pypi_0 | pypi |
| google-api-python-client | 2.64.0 | pypi_0 | pypi |
| google-auth | 2.22.0 | pypi_0 | pypi |
| google-auth-httplib2 | 0.1.0 | pypi_0 | pypi |
| googleapis-common-protos | 1.60.0 | pypi_0 | pypi |
| gunicorn | 20.1.0 | pypi_0 | pypi |
| gym | 0.26.2 | pypi_0 | pypi |
| gym-notices | 0.0.8 | pypi_0 | pypi |
| h11 | 0.14.0 | pypi_0 | pypi |
| httpcore | 0.17.3 | pypi_0 | pypi |
| httplib2 | 0.22.0 | pypi_0 | pypi |
| httpx | 0.24.1 | pypi_0 | pypi |
| huggingface-hub | 0.16.4 | pypi_0 | pypi |
| icetk | 0.0.4 | pypi_0 | pypi |
| idna | 3.4 | pypi_0 | pypi |
| imageio | 2.31.2 | pypi_0 | pypi |
| importlib-metadata | 6.8.0 | pypi_0 | pypi |
| importlib-resources | 5.10.4 | pypi_0 | pypi |
| ipykernel | 6.25.1 | pyh71e2992_0 | conda-forge |
| ipython | 8.15.0 | pyh0d859eb_0 | conda-forge |
| ipywidgets | 8.1.0 | pypi_0 | pypi |
| isort | 5.9.3 | pyhd3eb1b0_0 |
| itsdangerous | 2.1.2 | pypi_0 | pypi |
| jedi | 0.19.0 | pyhd8ed1ab_0 | conda-forge |
| jinja2 | 3.1.2 | pypi_0 | pypi |
| jmespath | 1.0.1 | pypi_0 | pypi |
| joblib | 1.3.2 | pypi_0 | pypi |
| jsonlines | 3.1.0 | pypi_0 | pypi |
| jsonschema | 4.19.0 | pypi_0 | pypi |
| jsonschema-specifications | 2023.7.1 | pypi_0 | pypi |
| jupyter_client | 7.3.4 | pyhd8ed1ab_0 | conda-forge |
| jupyter_core | 5.3.1 | py39hf3d152e_0 | conda-forge |
| jupyterlab-widgets | 3.0.8 | pypi_0 | pypi |
| kiwisolver | 1.4.5 | pypi_0 | pypi |
| langcodes | 3.3.0 | pypi_0 | pypi |
| lazy-object-proxy | 1.6.0 | py39h27cfd23_0 |
| ld_impl_linux-64 | 2.38 | h1181459_1 |
| libffi | 3.4.4 | h6a678d5_0 |
| libgcc-ng | 11.2.0 | h1234567_1 |
| libgomp | 11.2.0 | h1234567_1 |
| libsodium | 1.0.18 | h36c2ea0_1 | conda-forge |
| libstdcxx-ng | 11.2.0 | h1234567_1 |
| llvmlite | 0.39.1 | pypi_0 | pypi |
| lxml | 4.9.3 | pypi_0 | pypi |
| mako | 1.2.4 | pypi_0 | pypi |
| markupsafe | 2.1.3 | pypi_0 | pypi |
| matplotlib | 3.6.3 | pypi_0 | pypi |
| matplotlib-inline | 0.1.6 | pyhd8ed1ab_0 | conda-forge |
| mccabe | 0.7.0 | pyhd3eb1b0_0 |
| mediapipe | 0.10.3 | pypi_0 | pypi |
| moverscore | 1.0.3 | pypi_0 | pypi |
| mpmath | 1.3.0 | pypi_0 | pypi |
| multidict | 6.0.4 | pypi_0 | pypi |
| multiprocess | 0.70.13 | pypi_0 | pypi |
| murmurhash | 1.0.9 | pypi_0 | pypi |
| nbformat | 5.7.0 | pypi_0 | pypi |
| ncurses | 6.4 | h6a678d5_0 |
| nest-asyncio | 1.5.7 | pypi_0 | pypi |
| networkx | 3.1 | pypi_0 | pypi |
| nltk | 3.8.1 | pypi_0 | pypi |
| numba | 0.56.4 | pypi_0 | pypi |
| numpy | 1.25.2 | pypi_0 | pypi |
| open-clip-torch | 2.20.0 | pypi_0 | pypi |
| open3d | 0.17.0 | pypi_0 | pypi |
| openai | 0.28.0 | pypi_0 | pypi |
| opencv-contrib-python | 4.8.0.76 | pypi_0 | pypi |
| opencv-python | 4.8.0.76 | pypi_0 | pypi |
| openssl | 3.0.10 | h7f8727e_2 |
| outcome | 1.2.0 | pypi_0 | pypi |
| packaging | 23.1 | pyhd8ed1ab_0 | conda-forge |
| pandas | 2.1.0 | pypi_0 | pypi |
| param | 1.13.0 | pypi_0 | pypi |
| parameterized | 0.9.0 | pypi_0 | pypi |
| parso | 0.8.3 | pyhd8ed1ab_0 | conda-forge |
| pathy | 0.10.2 | pypi_0 | pypi |
| pexpect | 4.8.0 | pyh1a96a4e_2 | conda-forge |
| pickleshare | 0.7.5 | py_1003 | conda-forge |
| pillow | 10.0.0 | pypi_0 | pypi |
| pip | 23.2.1 | py39h06a4308_0 |
| platformdirs | 3.10.0 | pyhd8ed1ab_0 | conda-forge |
| plotly | 5.16.1 | pypi_0 | pypi |
| portalocker | 2.7.0 | pypi_0 | pypi |
| preshed | 3.0.8 | pypi_0 | pypi |
| prompt-toolkit | 3.0.39 | pyha770c72_0 | conda-forge |
| prompt_toolkit | 3.0.39 | hd8ed1ab_0 | conda-forge |
| protobuf | 3.20.3 | pypi_0 | pypi |
| psutil | 5.9.5 | pypi_0 | pypi |
| ptyprocess | 0.7.0 | pyhd3deb0d_0 | conda-forge |
| pure_eval | 0.2.2 | pyhd8ed1ab_0 | conda-forge |
| pyarrow | 11.0.0 | pypi_0 | pypi |
| pyasn1 | 0.5.0 | pypi_0 | pypi |
| pyasn1-modules | 0.3.0 | pypi_0 | pypi |
| pycares | 4.3.0 | pypi_0 | pypi |
| pycparser | 2.21 | pypi_0 | pypi |
| pyct | 0.5.0 | pypi_0 | pypi |
| pydantic | 1.10.12 | pypi_0 | pypi |
| pyemd | 0.5.1 | pypi_0 | pypi |
| pyext | 0.7 | pypi_0 | pypi |
| pygments | 2.16.1 | pyhd8ed1ab_0 | conda-forge |
| pyhocon | 0.3.60 | pypi_0 | pypi |
| pylint | 2.16.2 | py39h06a4308_0 |
| pymongo | 4.2.0 | pypi_0 | pypi |
| pyparsing | 3.0.9 | pypi_0 | pypi |
| pyquaternion | 0.9.9 | pypi_0 | pypi |
| pyrealsense2 | 2.54.1.5217 | pypi_0 | pypi |
| pysocks | 1.7.1 | pypi_0 | pypi |
| python | 3.9.17 | h955ad1f_0 |
| python-dateutil | 2.8.2 | pyhd8ed1ab_0 | conda-forge |
| python_abi | 3.9 | 2_cp39 | conda-forge |
| pytorch-pretrained-bert | 0.6.2 | pypi_0 | pypi |
| pytrec-eval | 0.5 | pypi_0 | pypi |
| pytz | 2023.3 | pypi_0 | pypi |
| pyyaml | 6.0.1 | pypi_0 | pypi |
| pyzmq | 25.1.0 | py39h6a678d5_0 |
| readline | 8.2 | h5eee18b_0 |
| referencing | 0.30.2 | pypi_0 | pypi |
| regex | 2023.8.8 | pypi_0 | pypi |
| requests | 2.31.0 | pypi_0 | pypi |
| responses | 0.18.0 | pypi_0 | pypi |
| retrying | 1.3.4 | pypi_0 | pypi |
| revchatgpt | 0.1.1 | pypi_0 | pypi |
| rouge-score | 0.1.2 | pypi_0 | pypi |
| rpds-py | 0.10.0 | pypi_0 | pypi |
| rsa | 4.9 | pypi_0 | pypi |
| s3transfer | 0.6.2 | pypi_0 | pypi |
| sacrebleu | 2.2.1 | pypi_0 | pypi |
| sacremoses | 0.0.53 | pypi_0 | pypi |
| safetensors | 0.3.3 | pypi_0 | pypi |
| scaleapi | 2.13.1 | pypi_0 | pypi |
| scikit-learn | 1.1.3 | pypi_0 | pypi |
| scipy | 1.9.3 | pypi_0 | pypi |
| seaborn | 0.11.2 | pypi_0 | pypi |
| selenium | 4.12.0 | pypi_0 | pypi |
| sentencepiece | 0.1.99 | pypi_0 | pypi |
| setuptools | 68.0.0 | py39h06a4308_0 |
| simple-slurm | 0.2.6 | pypi_0 | pypi |
| six | 1.16.0 | pyh6c4a22f_0 | conda-forge |
| smart-open | 6.3.0 | pypi_0 | pypi |
| sniffio | 1.3.0 | pypi_0 | pypi |
| sortedcontainers | 2.4.0 | pypi_0 | pypi |
| sounddevice | 0.4.6 | pypi_0 | pypi |
| soupsieve | 2.5 | pypi_0 | pypi |
| spacy | 3.5.4 | pypi_0 | pypi |
| spacy-legacy | 3.0.12 | pypi_0 | pypi |
| spacy-loggers | 1.0.4 | pypi_0 | pypi |
| sqlite | 3.41.2 | h5eee18b_0 |
| sqlitedict | 1.7.0 | pypi_0 | pypi |
| srsly | 2.4.7 | pypi_0 | pypi |
| stack_data | 0.6.2 | pyhd8ed1ab_0 | conda-forge |
| stanza | 1.5.0 | pypi_0 | pypi |
| summ-eval | 0.892 | pypi_0 | pypi |
| surge-api | 1.1.4 | pypi_0 | pypi |
| sympy | 1.11.1 | pypi_0 | pypi |
| tabulate | 0.9.0 | pypi_0 | pypi |
| tenacity | 8.2.3 | pypi_0 | pypi |
| thinc | 8.1.12 | pypi_0 | pypi |
| threadpoolctl | 3.2.0 | pypi_0 | pypi |
| tiktoken | 0.3.3 | pypi_0 | pypi |
| timm | 0.9.7 | pypi_0 | pypi |
| tk | 8.6.12 | h1ccaba5_0 |
| tls-client | 0.2.1 | pypi_0 | pypi |
| tokenizers | 0.13.3 | pypi_0 | pypi |
| tomli | 2.0.1 | py39h06a4308_0 |
| tomlkit | 0.11.1 | py39h06a4308_0 |
| torch | 1.12.1 | pypi_0 | pypi |
| torchvision | 0.13.1 | pypi_0 | pypi |
| tornado | 6.1 | py39hb9d737c_3 | conda-forge |
| tqdm | 4.66.1 | pypi_0 | pypi |
| traitlets | 5.9.0 | pyhd8ed1ab_0 | conda-forge |
| transformers | 4.33.1 | pypi_0 | pypi |
| trio | 0.22.2 | pypi_0 | pypi |
| trio-websocket | 0.10.3 | pypi_0 | pypi |
| typer | 0.9.0 | pypi_0 | pypi |
| typing | 3.7.4.3 | pypi_0 | pypi |
| typing-extensions | 4.7.1 | hd8ed1ab_0 | conda-forge |
| typing_extensions | 4.7.1 | pyha770c72_0 | conda-forge |
| tzdata | 2023.3 | pypi_0 | pypi |
| uncertainty-calibration | 0.1.4 | pypi_0 | pypi |
| undetected-chromedriver | 3.5.3 | pypi_0 | pypi |
| uritemplate | 4.1.1 | pypi_0 | pypi |
| urllib3 | 1.26.16 | pypi_0 | pypi |
| wasabi | 1.1.2 | pypi_0 | pypi |
| wcwidth | 0.2.6 | pyhd8ed1ab_0 | conda-forge |
| websocket-client | 1.3.3 | pypi_0 | pypi |
| websockets | 11.0.3 | pypi_0 | pypi |
| werkzeug | 2.2.3 | pypi_0 | pypi |
| wheel | 0.38.4 | py39h06a4308_0 |
| widgetsnbextension | 4.0.8 | pypi_0 | pypi |
| wrapt | 1.14.1 | py39h5eee18b_0 |
| wsproto | 1.2.0 | pypi_0 | pypi |
| xlrd | 2.0.1 | pypi_0 | pypi |
| xxhash | 3.3.0 | pypi_0 | pypi |
| xz | 5.4.2 | h5eee18b_0 |
| yarl | 1.9.2 | pypi_0 | pypi |
| zeromq | 4.3.4 | h9c3ff4c_1 | conda-forge |
| zipp | 3.16.2 | pypi_0 | pypi |
| zlib | 1.2.13 | h5eee18b_0 |
| zstandard | 0.18.0 | pypi_0 | pypi |
